src/pages/404.jsx

```diff
@@ -0,0 +1,14 @@
+import Head from 'next/head';
+import Link from 'next/link';
+
+export default function ErrorPage() {
+    return(
+
+        <div>
+        <Head>
+        <title>404 Not Found</title></Head>
+        <h1>404 Not Found</h1>
+        <Link href="/">Overview Page</Link>
```

Comment:
I think we should fix some of the formatting here: 
```
<div>
  <Head>
    <title>404 Not Found</title>
  </Head>
  <h1>404 Not Found</h1>
  <Link href="/">Overview Page</Link>
</div>
```

Reply:
Just fixed the formatting :)